index.js

```diff
@@ -77,21 +94,22 @@ genderCount(data);
 // ***************************************************************************
 // Bonus - Iteration 6
 // ***************************************************************************
-
 const promo20 = users => {
-  // Your code goes here ...
+  users.filter(item => item.balance.replace('$', "").replace(',', '')
```

Comment:
Same thing as above. a filter returns back a value. You might be console logging stuff, but the value of this function would be undefined. A filter is an array method that *always* returns back another array. 

And that array will be either the same length, or a smaller length than the original one, as long as the callback function you give to it returns back a truthy value.

In this case you are just doing a console log, which has no inherit value. If its not clear i can explain better directly :)